src/ui/app/App.js

```diff
@@ -182,6 +171,20 @@ import AboutPage from 'ui/about/AboutPage';
 import LicensePage from 'ui/license/LicensePage';
 import getRuntimeEnv from 'helpers/getRuntimeEnv';
 
+const EditProfileTypesPage = React.lazy(() => import('ui/profile-types/edit/EditProfileTypesPage'));
+const AddProfileTypesPage = React.lazy(() => import('ui/profile-types/add/AddProfileTypesPage'));
+const ListProfileTypePage = React.lazy(() => import('ui/profile-types/list/ListProfileTypePage'));
+const UserAuthorityPageContainer = React.lazy(() => import('ui/users/authority/UserAuthorityPageContainer'));
+const MyProfilePageContainer = React.lazy(() => import('ui/users/my-profile/MyProfilePageContainer'));
+const UserRestrictionsPage = React.lazy(() => import('ui/users/restrictions/UserRestrictionsPage'));
+const DetailUserPage = React.lazy(() => import('ui/users/detail/DetailUserPage'));
+const EditUserPage = React.lazy(() => import('ui/users/edit/EditUserPage'));
+const AddUserPage = React.lazy(() => import('ui/users/add/AddUserPage'));
+const UserListPage = React.lazy(() => import('ui/users/list/UserListPage'));
+
+export const renderWithSuspense = component =>
+  <Suspense fallback={<span>Loading..</span>}>{component}</Suspense>;
```

Comment:
do we have designs for a better loading screen?

Reply:
nope, but its loading almost instantly, maybe we can replace it with just a spinner

Reply:
nope, but its loading almost instantly, maybe we can replace it with just a spinner

Reply:
replaced with `Spinner`

Reply:
thanks!